### ISSUE TITLE
3D viewer: Menu missing

### ISSUE THREAD

**Issue by aschafu** [ODonoghueLab/Aquaria#4]
Sometimes the menu of the 3D viewer does not load. -- For the same entry, sometimes the menu comes, sometimes not.  
Browser: Firefox

**Comment by Traksewt** [comment]
Yes this is reproducable. The merging of GUI elements and the 3D Canvas has issues in Java3D. 

Temporary workaround: Click up the top of the canvas where the menu should be, and it will magically appear.
